campaigns.py

```diff
@@ -78,9 +78,9 @@ def deploy_banner(application):
     content = env['template_contents']
     put(StringIO.StringIO(content), remote_filename, use_sudo=True, mirror_local_mode=True)
     sudo('chown deploy:deploy %s' % remote_filename)
+    execute(app.restart, application)
```

Comment:
I'm vaguely wondering whether it's even worth having a conditional in here to deal with that. I think the only thing that frontend serves that would error for 15 seconds is search, everything else is entirely static and would be served by the mirrors. Happy to go either way.